### DISCUSSION TITLE
Play a tape through my normal terminal

### DISCUSSION THREAD

**Discussion by 0atman** [charmbracelet/vhs#228] [Ideas]
I'd like to play a tape through my normal terminal (or, in fact, screen capture it myself inside cool-retro-term).

Is it possible to do this? Thanks!

**Comment by rm-dr** [comment]
Not yet, but maybe later. Probably not happening soon, though.

It shouldn't be hard to write a small script that reads tape files and executes commands. You could also write a shell script instead of a tape file, of course.

**Comment by bashbunni** [comment]
This will definitely add to the support/compatibility burden for VHS. I know we're using xterm for simplicity and control of the terminal environment across machines. I'm going to convert this to a discussion for the time being, and if we decide to go ahead with this, we can create a new issue from the discussion.
